open-machine-learning-jupyter-book/assignments/ml-advanced/unsupervised-learning/unsupervised-intro-with-kmeans-clustering.ipynb

```diff
@@ -0,0 +1,674 @@
+{
```

Comment:
<p>we don't load the dataset twice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
ok